string ClassName
float32 conf
int32 tl_x
int32 tl_y
int32 br_x
int32 br_y
